api/grpcserver/grpc_server.go

```
@@ -63,7 +68,7 @@ func (s *GrpcServer) GetAllUsersLikeUsername(_ context.Context, username *proto.
 	for rows.Next() {
 		u := proto.User{}
 
-		rows.Scan(&u.UserName, &u.RealName, &u.Bio, &u.AvatarUrl)
+		rows.Scan(&u.Id, &u.UserName, &u.RealName, &u.Bio, &u.AvatarUrl)
```

you can just return the err if `rows.Scan` returns a non nil err (like below)